Sources/TuistKit/Commands/HashCommand.swift

```diff
@@ -0,0 +1,36 @@
+import ArgumentParser
+import Foundation
+import TuistSupport
+
+/// A command to hash an Xcode or generated project.
+public struct HashCommand: AsyncParsableCommand {
```

Comment:
I don't think this needs to be public if we don't need to interact with in the cli ext